src/Servers/Kestrel/Core/src/Internal/Http2/Http2Connection.Generated.tt

```diff
@@ -0,0 +1,67 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
```

Comment:
I don't think we should use things that don't work on linux.

Reply:
I used it as Visual Studio develop-time generator. It's not hooked up in the project-file, so the .tt has no effect during other bilds?
It's aligned to https://github.com/aspnet/AspNetCore/blob/master/src/Servers/HttpSys/src/RequestProcessing/RequestHeaders.Generated.tt which is also not in the csproj.

Would it be better to place this code-generation in https://github.com/aspnet/AspNetCore/tree/master/src/Servers/Kestrel/tools/CodeGenerator?

Reply:
Yep

Reply:
Ah. I was hoping this did work on linux. Good catch.

Reply:
Changed with 5ac5b63efdd16a7e931d223452195a169a3172f9